client/state/themes/actions.js

```diff
@@ -765,3 +768,44 @@ export function confirmDelete( themeId, siteId ) {
 		);
 	};
 }
+
+function setPreviewingThemeData( themeId, siteId ) {
+	return {
+		type: PREVIEWING_THEME_DATA,
+		themeData: {
+			themeId,
+			siteId
```

Comment:
Actually, I think we shouldn't pass `siteId` as an arg to any theme preview related action at all, simply because conceptually, the preview doesn't need a `siteId`. (This is visible from the fact that we don't ever show a site selector _before_ the preview, only after.)

I realize we need it in practice here for [`getTheme`](https://github.com/Automattic/wp-calypso/pull/11167/files#diff-8973cec3a24a1ea3d7e46f0a0543e7a8R114) (to obtain a `theme` object, which we then need to `getPreviewUrl`). However, the only information we need is the `demo_uri`, and that is always stored either in the `wpcom` subtree, or, for JP sites, _if_ the theme is on WP.org, in the `wporg` one (and added by us to the theme object returned by `getTheme` for a JP site).

I think what would be cleanest would be to use a `source` arg to the that can be either `wpcom` or `wporg` (and use that then as `getTheme`'s `siteId`).